concise-mid-tag.cddl

```diff
@@ -174,7 +174,10 @@ tagged-svn = #6.552(svn)
 tagged-min-svn = #6.553(min-svn)
 svn-type-choice = tagged-svn / tagged-min-svn
 
-flags-type = bytes .bits $operational-flags
+flags-type = {
```

Comment:
Are code point definitions needed?

Reply:
Should a value of 1 in the mask expose the flag or mask it off?

Reply:
> Are code point definitions needed?

no, but i got it wrong, it should have `[`s rather than `{`s

> Should a value of 1 in the mask expose the flag or mask it off?

I'd say expose

Reply:
now with the square brackets it'll encode to, e.g.:
```
[h'3E', h'FF']
```
or
```
[h'2B']
```